MBBSEmu.Tests/ExportedModules/Majorbbs/sortstgs_Tests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using MBBSEmu.Memory;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class sortstgs_Tests : ExportedModuleTestBase
+    {
+        private const int SORTSTGS_ORDINAL = 558;
+        
+        [Theory]
+        [InlineData(new[] { "This", "is", "an", "array" }, new[] { "an", "array", "is", "This" })]
+        [InlineData(new[] { "Hello", "my", "name", "is" }, new[] { "Hello", "is", "my", "name" })]
+        [InlineData(new[] { "Hello", "Anger" }, new[] { "Anger", "Hello" })]
+        [InlineData(new[] { "zzzz", "BBBB", "CCCC", "aaaa", "000AAA" }, new[] { "000AAA", "aaaa", "BBBB", "CCCC", "zzzz" })]
+        [InlineData(new[] { "zzzz", "BBBB", "CCCC", "aaaa", "000AAA", "111ZZZ", "!!!XXX" }, new[] { "!!!XXX", "000AAA", "111ZZZ", "aaaa", "BBBB", "CCCC", "zzzz" })]
+        public void sortstgs_Test(string[] inputArray, string[] expectedArray)
+        {
+            //Reset State
+            Reset();
+
+            //Set Argument Values to be Passed In
+            var stringPointer = new IntPtr16[inputArray.Length];
+
+            for (var i = 0; i < inputArray.Length; i++)
+                stringPointer[i] = mbbsEmuMemoryCore.AllocateVariable("INPUT_STRING"+ i, (ushort)(inputArray[i].Length + 1));
```

Comment:
nit: add extra space after the string and before the `+` sign, i.e.

`"INPUT_STRING" + i`